packages/filefeeds/src/config.ts

```diff
@@ -73,6 +73,11 @@ export interface FileFeedsParams {
    */
   userJwt: string
 
+  /**
+   * The session token to use when launching OneSchema FileFeeds.
+   */
+  sessionToken?: string
+
```

Comment:
nit: Would be great to have this item before any other, since it takes precedence when both are provided.